docs/source/gateway/index.rst

```diff
@@ -1,6 +1,780 @@
+provider services by click on the "try it now" option within the endpoint definition entry.
+
+Step 6: Send Requests to Routes via REST API
+--------------------------------------------
+You can now send requests to the exposed routes.
+See the :ref:`REST examples <gateway_rest_api>` for guidance on request formatting.
+
+Step 7: Send Requests Using the Fluent API
+------------------------------------------
+
+Here's an example of how to send a chat request using the ``fluent`` API:
+
+.. code-block:: python
+
+    from mlflow.gateway import query, set_gateway_uri
+
+    set_gateway_uri(gateway_uri="http://localhost:5000")
+
+    response = query(
+        "chat",
+        {"messages": [{"role": "user", "content": "What is the best day of the week?"}]},
+    )
+
+    print(response)
+
+**Note:** Remember to change the uri definition to the actual uri of your Gateway server.
+
+The returned response will be in this data structure (the actual content and token values will likely be different):
+
+.. code-block:: python
+
+    {
+        "candidates": [
+            {
+                "message": {
+                    "role": "assistant",
+                    "content": "\n\nIt's hard to say what the best day of the week is.",
+                },
+                "metadata": {"finish_reason": "stop"},
+            }
+        ],
+        "metadata": {
+            "input_tokens": 13,
+            "output_tokens": 15,
+            "total_tokens": 28,
+            "model": "gpt-3.5-turbo-0301",
+            "route_type": "llm/v1/chat",
+        },
+    }
+
+
+Step 8: Send Requests Using the Client API
+------------------------------------------
+See the :ref:`MLflowGatewayClient <gateway_client_api>` section for further information.
+
+.. _gateway-concepts:
+
+Concepts
+========
+
+There are several concepts that are referred to within the MLflow AI Gateway APIs, the configuration definitions, examples, and documentation.
+Becoming familiar with these terms will help in configuring new endpoints (routes) and ease the use of the interface APIs for the AI Gateway.
+
+.. _providers:
+
+Providers
+---------
+The MLflow AI Gateway is designed to support a variety of model providers.
+A provider represents the source of the machine learning models, such as OpenAI, Anthropic, and so on.
+Each provider has its specific characteristics and configurations that are encapsulated within the model part of a route in the MLflow AI Gateway.
+
+Supported Provider Models
+~~~~~~~~~~~~~~~~~~~~~~~~~
+The table below presents a non-exhaustive list of models and a corresponding route type within the MLflow AI Gateway.
+With the rapid development of LLMs, there is no guarantee that this list will be up to date at all times. However, the associations listed
+below can be used as a helpful guide when configuring a given route for any newly released model types as they become available with a given provider.
+
+.. list-table::
```

Comment:
<img width="1009" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/9553bb2b-3fd3-4169-990e-dbeca4c4cd04">

✓ and ✗ almost look the same. Can we use differenct icons or yes and no?

Reply:
I didn't want to introduce the emoji dependency into sphinx (I'm terrified of how unstable it might be for the build process) so I needed to use the Unicode characters, which I agree look a little hard to read with our CSS. 
I'll change to "yes" and "no" instead.

Reply:
yes and no look great